[mega branch] main spring launch event PR (#5467)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes.md

  - Loading the Jaffle Shop sample data (provided by dbt Labs) into Azure Synapse Analytics. 
  - Connecting dbt Cloud to Azure Synapse Analytics.
  - Turning a sample query into a model in your dbt project. A model in dbt is a SELECT statement.
  - Adding tests to your models.
  - Documenting your models.
  - Scheduling a job to run.

  </expandable>

- **New:** [dbt Assist](/docs/cloud/dbt-assist) is a powerful AI feature, available in beta, helping you generate documentation, metrics, semantic models, and tests, saving you time and helping you deliver data that works. Available for dbt Cloud Enterprise accounts in the [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud).

- **New:** The VS Code extension [Power user for dbt Core and dbt Cloud](https://docs.myaltimate.com/arch/beta/) is now available in beta for [dbt Cloud CLI](/docs/cloud/cloud-cli-installation#vs-code-extension) users. The extension accelerates dbt and SQL development and includes features such as generating models from your source definitions or SQL, updating dbt model using natural language, and [more](https://docs.myaltimate.com/)!

> - **New:** MetricFlow enables you to now add [metrics as dimensions](/docs/build/ref-metrics-in-filters) to your metric filters to create more complex metrics and gain more insights. Available for all users.

Author: ```suggestion
- **New:** MetricFlow enables you to now add metrics as dimensions to your metric filters to create more complex metrics and gain more insights. Available for all users.
```

  dbt Explorer's current capabilities are now Generally Available for dbt Cloud customers. With Explorer, you can more easily navigate your dbt Cloud project – including models, sources, and their columns – to gain a better understanding of its latest production or staging state.

  To learn more about its features, check out:
  
  - [Explore projects](/docs/collaborate/explore-projects)
  - [Explore multiple projects](/docs/collaborate/explore-multiple-projects) 
  - [Column-level lineage](/docs/collaborate/column-level-lineage) 
  - [Model performance](/docs/collaborate/model-performance) 
  - [Project recommendations](/docs/collaborate/project-recommendations) 

  </expandable>

- **New:** Native support for Microsoft Fabric in dbt Cloud is now GA. This feature is powered by the [dbt-fabric](https://github.com/Microsoft/dbt-fabric) adapter. To learn more, refer to [Connect Microsoft Fabric](/docs/cloud/connect-data-platform/connect-microsoft-fabric) and [Microsoft Fabric DWH configurations](/reference/resource-configs/fabric-configs). There's also a [quickstart guide](https://docs.getdbt.com/guides/microsoft-fabric?step=1) to help you get started. 

> - **New:** [dbt Mesh](https://docs.getdbt.com/best-practices/how-we-mesh/mesh-1) is now GA to dbt Cloud Enterprise users. dbt Mesh is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections. Get started with dbt Mesh by reading the [dbt Mesh quickstart guide](https://docs.getdbt.com/guides/mesh-qs?step=1).

Reviewer: Fixing to fix failing test
```suggestion
- **New:** dbt Mesh is now GA to dbt Cloud Enterprise users. dbt Mesh is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections. Get started with dbt Mesh by reading the [dbt Mesh quickstart guide](https://docs.getdbt.com/guides/mesh-qs?step=1).
```

> - **New:** [dbt Mesh](ttps://docs.getdbt.com/best-practices/how-we-mesh/mesh-1-) is now GA to dbt Cloud Enterprise users. dbt Mesh is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections. Get started with dbt Mesh by reading the [dbt Mesh quickstart guide](https://docs.getdbt.com/guides/mesh-qs?step=1).

Author: ```suggestion
- **New:** [dbt Mesh](https://docs.getdbt.com/best-practices/how-we-mesh/mesh-1-) is now GA to dbt Cloud Enterprise users. dbt Mesh is a framework that helps organizations scale their teams and data assets effectively. It promotes governance best practices and breaks large projects into manageable sections. Get started with dbt Mesh by reading the [dbt Mesh quickstart guide](https://docs.getdbt.com/guides/mesh-qs?step=1).
```

- <expandable alt_header="New: Preview of native support for Azure Synapse Analytics" > 

  Native support in dbt Cloud for Azure Synapse Analytics is now available in [Public Preview](/docs/dbt-versions/product-lifecycles#dbt-cloud)!

  To learn more, refer to [Connect Azure Synapse Analytics](/docs/cloud/connect-data-platform/connect-azure-synapse-analytics) and [Microsoft Azure Synapse DWH configurations](/reference/resource-configs/azuresynapse-configs).

  Also, check out the [Quickstart for dbt Cloud and Azure Synapse Analytics](/guides/azure-synapse-analytics?step=1). The guide walks you through:

  - Loading the Jaffle Shop sample data (provided by dbt Labs) into Azure Synapse Analytics. 
  - Connecting dbt Cloud to Azure Synapse Analytics.
  - Turning a sample query into a model in your dbt project. A model in dbt is a SELECT statement.
  - Adding tests to your models.
  - Documenting your models.
  - Scheduling a job to run.

  </expandable>

- **New:** MetricFlow enables you to now add metrics as dimensions to your metric filters to create more complex metrics and gain more insights. Available for all dbt Semantic Layer users.

> - **New:** [Staging environment](/docs/deploy/deploy-environments#staging-environment) is now GA. Use staging environments to grant developers access to deployment workflows and tools while controlling access to production data. Available to dbt Cloud users.

Author: ```suggestion
- **New:** [Staging environment](/docs/deploy/deploy-environments#staging-environment) is now GA. Use staging environments to grant developers access to deployment workflows and tools while controlling access to production data. Available to all users.
```
